text/0000-ibm-liberty-buildpack.md

```diff
@@ -0,0 +1,68 @@
+# Paketo Community IBM Liberty Buildpack
+
+## Summary
+
+1. Create a Cloud Native Buildpack for the purpose of installing the IBM [OpenLiberty](https://openliberty.io/) Java runtime. 
+2. Create a new top-level multi buildpack that packages up Eclipse OpenJ9 & OpenLiberty for use by application developers.
+
+## Motivation
+
+A [ibm-websphere-liberty-buildpack](https://github.com/cloudfoundry/ibm-websphere-liberty-buildpack) exists in the cloudfoundry community and is widely used and popular.  This RFC will
+take that buildpack to the next level, cloud native.  This buildpack will eventually replace the v2 CF ibm-websphere-buildpack.   
+
+## Detailed Explanation
+
+### IBM OpenLiberty Buildpack
+
+This buildpack has the purpose of installing the IBM Open Liberty. It will will be based on the Paketo [apache-tomcat](https://github.com/paketo-buildpacks/apache-tomcat) buildpack. 
+
+This buildpack will participate if all the following conditions are met:
+
+- `<APPLICATION_ROOT>/WEB-INF` exists or
+-  A liberty server directory exists or
+-  A liberty packaged server exists and
```

Comment:
In what cases would we encounter a packaged server? If you pass a archive to `pack build` or most other platforms the platform will add the extracted contents to the app dir. Also, our existing build tool buildpacks look for an artifact after building and replace the source code with the expanded artifact. Therefore, I would image we would always encounter an expanded server rather than a packaged one?

Reply:
The packaged server is similar to the server directory.   The contents of the archive would be similar to the contents of the server directory.   The platform extracting the archive is actually helpful.   The buildpack would just need to copy the server.xml and app to the location where the liberty server expects. 

This is not yet implemented and in order to expedite the delivery of the buildpack it may have to be delivered in the future.